CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [1.1.0] - 2020-08-05
+
+### Changes
+- Fix readMe to add the minimum minor version supported by sdk. 1.11 to 1.11.4
```

Comment:
1.1.0 is repeated twice....move this log into the already created changes section for 1.1.0